infrastructure/environments/cloudformation/full/common/lambda/ecs_scale/tests/test_handler.py

```diff
@@ -0,0 +1,49 @@
+import os
+import unittest
+from unittest.mock import patch, Mock, call, MagicMock
+from ecs_scale_down.handler import lambda_handler
+import logging
+
+
+class TestLambdaHandler(unittest.TestCase):
+
+    @patch("os.environ.get")
+    @patch("boto3.client")
+    def test_lambda_handler(self, mock_boto_client, mock_get_env):
+        # Mock environment variables
+        mock_get_env.side_effect = (
+            "my-cluster",  # Return 'my-cluster' for 'ECS_CLUSTER_NAME'
+            "dagit-service",  # Return 'dagit-service' for 'ECS_DAGIT_SERVICE_NAMES'
+            "daemon-service",  # Return 'daemon-service' for 'ECS_DAEMON_SERVICE_NAMES'
+            "code-server-service",  # Return 'code-server-service' for 'ECS_CODE_SERVER_SERVICE_NAMES'
+        )
+
+        mock_ecs_client = Mock()
+        mock_boto_client.return_value = mock_ecs_client
+
+        # Mock ecs_client.update_service with a Mock object
+        mock_update_service = Mock(return_value={"some": "response"})
+
+        # Invoke the Lambda function with a sample event (optional)
+        event = {"key": "value"}  # You can provide a sample event here if needed
```

Comment:
Does the `pipelinestatus` default to "false" here? Just curious how in the `update_serice.assert_has_calls` the desiredCount is set to 0?

Reply:
Great point! I hadn't updated the tests. Will fix now.